src/fsharp/FSharp.DependencyManager.Nuget/README.md

```diff
@@ -0,0 +1,18 @@
+# `dotnet fsi`: NuGet Dependency Manager Plugin
+
+This extensions ship by default since .NET 5. It can be used in fsharp interactive through `dotnet fsi` without further setup through `#r "nuget:"` references.
```

Comment:
```suggestion
This extension ships by default since .NET 5. It can be used in F# interactive through `dotnet fsi` without further setup through `#r "nuget:"` references.
```

Reply:
thanks @cartermp!